.changeset/metal-guests-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@web3-ui/hooks': minor
+---
+
+The `hooks` package now has two new hooks: `useReadOnlyContract` which can be used to read from contracts without a signer and `useStaticProvider`.
```

Comment:
Let's elaborate on the use as from what I understand the latter facilitates the former here